policy/scan/local_scanner.go

```diff
@@ -395,6 +395,12 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 		}
 	}
 
+	// ensure that the root asset has keept his name from --asset-name
+	if job.Inventory.Spec.Assets[0].Name != assets[0].asset.Name {
```

Comment:
i dont think we can guarantee the inventory's asset ordering here. should just attaching this to the `cliAsset` not be enough?